Source/WebCore/svg/graphics/filters/SVGFilter.cpp

```diff
@@ -277,6 +273,33 @@ RefPtr<FilterImage> SVGFilter::apply(FilterImage* sourceImage, FilterResults& re
     return stack.takeLast();
 }
 
+Vector<FilterEffectStyle> SVGFilter::createStyles(const Filter&) const
+{
+    return createStyles();
+}
+
+Vector<FilterEffectStyle> SVGFilter::createStyles() const
```

Comment:
How does this work when the graph is not just a list? We can't render non-linear graphs with native filters.

Reply:
This function should not be reached. The FilterMode::GraphicsContext will be added by the FilterEffects which take only one input. I added the following assertion to this function:

```
ASSERT(supportedFilterModes().contains(FilterMode::GraphicsContext));
```